### DISCUSSION TITLE
Is there any way to customize the yAxisLabelTextsStyle?

### DISCUSSION THREAD

**Discussion by Ved-Ravi** [Abhinandan-Kushwaha/react-native-gifted-charts#880] [Q&A] [answered]
I need to change the font Family of y axis Label text. Is there any way to customize the yAxisLabelTextsStyle?

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Yes @Ved-Ravi you can use the `yAxisTextStyle` prop.
